jena-fuseki2/jena-fuseki-ui/package.json

```
@@ -60,7 +62,7 @@
     "sass": "^1.69.4",
     "sinon": "^17.0.0",
     "vite": "^5.0.12",
-    "vite-plugin-istanbul": "6.0.0",
+    "vite-plugin-istanbul": "6.0.2",
```

Maybe put back the `^` which was removed in the #2457 quick-fix.

```suggestion
    "vite-plugin-istanbul": "^6.0.2",
```
At this point in time it resolves to 6.0.2 and the yarn.lock file only changes to reflect adding the `^`.

---

Missed that, sorry. It's back now :+1: